src/XenCert/StorageHandler.py

```diff
@@ -2138,24 +2136,25 @@ def Create(self, device_config = {}):
                 device_config['targetIQN'] = '*'
             else:
                 device_config['targetIQN'] = self.iqn
-            if self.storage_conf['chapuser']!= None and self.storage_conf['chappasswd'] != None:
+            if self.storage_conf['chapuser']is not None and self.storage_conf['chappasswd'] is not None:
                 device_config['chapuser'] = self.storage_conf['chapuser']
                 device_config['chappassword'] = self.storage_conf['chappasswd']
             # try to create an SR with one of the LUNs mapped, if all fails throw an exception
             for scsiId in listSCSIId:
                 try:                    
                     device_config['SCSIid'] = scsiId
-                    XenCertPrint("The SR create parameters are %s, %s" % (util.get_localhost_uuid(self.session), device_config))
+                    device_config_tmp = XenCertCommon.getNoPasswordConfig(device_config, self.storage_conf['storage_type'])
+                    XenCertPrint("The SR create parameters are %s, %s" % (util.get_localhost_uuid(self.session), device_config_tmp))
                     sr_ref = self.session.xenapi.SR.create(util.get_localhost_uuid(self.session), device_config, '0', 'XenCertTestSR', '', 'lvmoiscsi', '',True, {})
-                    XenCertPrint("Created the SR %s using device_config %s" % (sr_ref, device_config))
+                    XenCertPrint("Created the SR %s using device_config %s" % (sr_ref, device_config_tmp))
```

Comment:
Just a suggestion: needless to print device_config_tmp again here.
    XenCertPrint("Created the SR %s" % sr_ref )